src/phantasm-renderer/backend/vulkan/Swapchain.hh

```diff
@@ -28,7 +30,7 @@ public:
 
     /// present the active backbuffer to the screen
     /// can trigger a resize instead of presenting if the swapchain is stale
-    void present();
+    bool present();

```

Comment:
what is the return value?

Reply:
Vulkan presents can fail and trigger a resize instead, this returns false in that case, i've added a comment explaining this.